src/renderer/screens/DatabaseScreen/MainView.tsx

```diff
@@ -55,61 +78,36 @@ export default function MainView() {
           onClick: () => {
             if (additionalData) {
               setTabs(tabs.filter((tab) => tab.key === additionalData.key));
+              setSelectedTab(additionalData.key);
             }
           },
         },
         {
           text: 'Close to the Right',
```

Comment:
- For `Close to the right`: Disable if there is nothing to the right
- For `Close other`: Disable when there is only one tab left